src/sonic-yang-models/yang-models/sonic-crm.yang

```diff
@@ -353,6 +353,46 @@ module sonic-crm {
                     type threshold;
                 }
 
+                leaf mpls_inseg_threshold_type {
```

Comment:
Please add tests for MPLS CRM.

Reply:
@kperumalbfn I looked at the existing CRM tests in sonic-yang-models.  It appears that there exist currently generic parameter checking that is applicable for *all* CRM types.  I don't think there needs to be anything specific for MPLS, since it can already make use of the generic CRM tests.

Reply:
@kperumalbfn I have added CRM tests for the 2 MPLS CRM types as requested.